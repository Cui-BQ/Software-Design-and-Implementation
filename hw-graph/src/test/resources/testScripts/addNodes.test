# Test addNode method in a single graph

# Create a graph
CreateGraph graph1

# Add different nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add a duplicated node
AddNode graph1 n1

# List its node
ListNodes graph1

# List child for n1
ListChildren graph1 n1

# Check contains() method.
Contains graph1 n1
Contains graph1 n4


