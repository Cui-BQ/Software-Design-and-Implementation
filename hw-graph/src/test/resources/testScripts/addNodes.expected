# Test addNode method in a single graph

# Create a graph
created graph graph1

# Add different nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add a duplicated node
failed to add node n1 to graph1

# List its node
graph1 contains: n1 n2 n3

# List child for n1
the children of n1 in graph1 are:

# Check contains() method.
graph1 contains n1: true
graph1 contains n4: false


